tests/test.dispensary.js

```diff
@@ -157,10 +200,26 @@ describe('Dispensary', function() {
       });
   });
 
+  it('should return cached libraries after first call to getLibraries', () => {
+    var dispensary = new Dispensary();
+
+    fsSpy.reset();
+
+    dispensary.getLibraries()
```

Comment:
Yeah, I just meant once I added the return it was still passing, yay. (I just haven't pushed yet as I'm fixing up the other issues before I rebase+push)